      fix refrence bug          In torch==1.5, the import of the API has changed. Although it does not interrupt the operation of the program, it seems to me to be an implicit error and may throw an exception in later versions. 
 import torch.nn.functional as F  import torch.optim as optim  import torch.optim.lr_scheduler as lr_scheduler import torch.utils.data  from torch.utils.tensorboard import SummaryWriter    import test  # import test.py to get mAP after each epoch 
